examples/filtering-ticket-groups/main.css

```diff
@@ -0,0 +1,58 @@
+body {
+    font-family: "Open Sans", sans-serif;
+}
+
+#container {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+#filters {
+    width: 100%;
+    display: flex;
+    list-style-type: none;
+    justify-content: center;
+}
+#filters li {
```

Comment:
I know it's just an example, but could you add spaces between these rules? It's a little hard on my eyes